src/client/src/rt-theme/globals.ts

```diff
@@ -54,10 +54,12 @@ export default injectGlobal`
     font-size: 16px;
     line-height: 1rem;
     text-rendering: geometricPrecision;
+    height: 100%;
+    overflow: auto;
   }
-  
-  html {
-    overflow-y: auto;
+
+  #root {
+    height: 100%;
```

Comment:
When resolving the merge conflict with #854 I believe this will perform better: https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/ARTP-250/add-styleguide/src/client/src/rt-theme/globals.ts#L68-L72

Which areas did this resolve problem? Or, could you test and verify that your problems are resolved by the version of `globals` on #854?

Reply:
The goal was to allow pop out components to use full width/height of their window. Please overwrite if you're able to get the same functionality with your approach.